ui/puzzle/src/ctrl.ts

```diff
@@ -501,8 +501,8 @@ export default function (opts: PuzzleOpts, redraw: Redraw): Controller {
     if (tree.nodeAtPath(path)?.puzzle == 'fail' && vm.mode != 'view') return;
     withGround(g => g.selectSquare(null));
     jump(path);
-    lichess.sound.saySan(vm.node.san, true);
     lichess.sound.move(vm.node);
+    lichess.sound.say('Is not the move, try something else!');
```

Comment:
Do you confirm it work as intended when you test it? Given the `if` above this seem it will also read it out when not failing